spec.bs

```diff
@@ -4582,6 +4607,20 @@ They return a {{double}}.
     participant metrics/code fetch time averager=].
 1. If |signalBaseValue| is "<code>[=signal base value/participating-ig-count=]</code>":
   1. Return |bidAndScoreMetrics|'s [=per participant metrics/participating interest group count=].
+1. If |signalBaseValue| is "<code>[=signal base value/percent-scripts-timeout=]</code>":
+  1. Return the result of [=computing a percentage metric=] given |metrics|'s [=per participant
+    metrics/script timeouts occurred=] and |metrics|'s [=per participant metrics/script executions
+    attempted=].
+
+</div>
+
+<div algorithm>
+To <dfn>compute a percentage metric</dfn> given {{long}}s |numerator| and |denominator|:
+1. If |denominator| is 0, return 0.
+1. Let |result| be 100.0 * |numerator| / |denominator|, performing the computation with
+  {{double}}s.
+1. If |result| &gt; 110.0, set |result| to 110.0

```

Comment:
is it useful to add a note about this 110 cap? Basically why we need this cap, and also why this number (if different implementations can choose their own cap, we can say it's a implementation defined cap).

Reply:
Done. A bit forward-looking, but, well, probably not worth worrying about referring to the very next CL in the chain.